From 95d7ce5d66e9a583045ce2789813a49e738b20eb Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Wed, 1 Nov 2017 14:49:05 -0500
Subject: [PATCH 3/3] Changed cipher specification to openssl format

By default, the config file is set up to accept ciphers in erlang format.
Our existing services (haproxy, lighttpd) both uses openssl format.
This change makes the config file generated by packstack assume the ciphers
given is in openssl format. Specifying ciphers in the same format makes it
easier to maintain.
---
 SPECS/puppet-rabbitmq.spec | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/SPECS/puppet-rabbitmq.spec b/SPECS/puppet-rabbitmq.spec
index c96099b..6541bc7 100644
--- a/SPECS/puppet-rabbitmq.spec
+++ b/SPECS/puppet-rabbitmq.spec
@@ -22,6 +22,7 @@ URL:            https://github.com/puppetlabs/puppetlabs-rabbitmq
 
 Source0:        https://github.com/puppetlabs/%{upstream_name}/archive/%{commit}.tar.gz#/%{upstream_name}-%{shortcommit}.tar.gz
 Patch0001:      0001-Roll-up-TIS-patches.patch
+Patch0002:      0002-Changed-cipher-specification-to-openssl-format.patch
 
 BuildArch:      noarch
 
@@ -35,6 +36,7 @@ Installs, configures, and manages RabbitMQ.
 %prep
 %setup -q -n %{tarsources}-%{upstream_version}
 %patch0001 -p1
+%patch0002 -p1
 
 find . -type f -name ".*" -exec rm {} +
 find . -size 0 -exec rm {} +
-- 
1.8.3.1

